hooked/views.py

```diff
@@ -63,7 +64,7 @@ def check_webhook_signature(self, request):
         match we just log an error.)
         """
         app_key = getattr(settings, APP_KEY_SETTING)
-        fail_on_mismatch = getattr(settings, FAIL_ON_MISMATCH_SETTING, False)
+        fail_on_mismatch = getattr(settings, FAIL_ON_MISMATCH_SETTING, True)
```

Comment:
We should document this, the README currently says that this defaults to `False`... also I'm still suspicious that maybe I broke something here and we should hold off on doing this